Intersect.Client/Entities/Player.cs

```diff
@@ -1415,16 +1415,21 @@ public void AutoTarget()
                 return;
             }
 
-            if (mlastTargetList.ContainsKey(currentEntity))
+            if(!Globals.Entities.TryGetValue(currentEntity.Id, out var targetedEntity))
             {
-                mlastTargetList[currentEntity].LastTimeSelected = Timing.Global.Milliseconds;
+                return;
+            }
+
+            if (mlastTargetList.TryGetValue(targetedEntity, out var _))
+            {
+                mlastTargetList[targetedEntity].LastTimeSelected = Timing.Global.Milliseconds;
```

Comment:
You are supposed to use the out variable here not look it up again in the target list.

Reply:
like this?
```
            if (mlastTargetList.TryGetValue(targetedEntity, out var lastTarget))
            {
                lastTarget.LastTimeSelected = Timing.Global.Milliseconds;
            }
```

Reply:
Yes @WeylonSantana